{"name": "test_epoch_length", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '/home/martink/Source/repos/cardano-node-tests2/tmp/testrun_20201102.log'", "trace": "tmp_path_factory = TempPathFactory(_given_basetemp=PosixPath('/tmp/pytest-of-martink/pytest-2/popen-gw3'), _trace=<pluggy._tracing.TagTracerSub object at 0x7f9bcd91cd90>, _basetemp=PosixPath('/tmp/pytest-of-martink/pytest-2/popen-gw3')), worker_id = 'gw3'\nrequest = <SubRequest 'cluster_manager' for <Function test_epoch_length>>\n\n    @pytest.yield_fixture\n    def cluster_manager(\n        tmp_path_factory: TempdirFactory,\n        worker_id: str,\n        request: FixtureRequest,\n    ) -> Generator[parallel_run.ClusterManager, None, None]:\n>       cluster_manager_obj = parallel_run.ClusterManager(\n            tmp_path_factory=tmp_path_factory, worker_id=worker_id, request=request\n        )\n\n/home/martink/Source/repos/cardano-node-tests2/cardano_node_tests/tests/conftest.py:121: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/home/martink/Source/repos/cardano-node-tests2/cardano_node_tests/utils/parallel_run.py:67: in __init__\n    self.lock_log = self._init_log()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <cardano_node_tests.utils.parallel_run.ClusterManager object at 0x7f9bcd3b7880>\n\n    def _init_log(self) -> Path:\n        \"\"\"Return path to run log file.\"\"\"\n        env_log = os.environ.get(\"SCHEDULING_LOG\")\n        if env_log:\n            run_log = Path(env_log).expanduser()\n            if not run_log.is_absolute():\n                # the path is relative to LAUNCH_PATH (current path can differ)\n                run_log = helpers.LAUNCH_PATH / run_log\n            # create the log file if it doesn't exist\n>           open(run_log, \"a\").close()\nE           FileNotFoundError: [Errno 2] No such file or directory: '/home/martink/Source/repos/cardano-node-tests2/tmp/testrun_20201102.log'\n\n/home/martink/Source/repos/cardano-node-tests2/cardano_node_tests/utils/parallel_run.py:82: FileNotFoundError"}, "description": "Test the \"epochLength\" configuration.", "attachments": [{"name": "log", "source": "85c8633a-51fe-480c-97f8-175125792ce5-attachment.txt", "type": "text/plain"}], "start": 1604310980586, "stop": 1604310980586, "uuid": "a47bc231-e480-4324-93e8-f9527eb79e5c", "historyId": "b9920589ae8db5055382d60ae0547102", "testCaseId": "9143b3fe87b2034557a7b9d1c0cf3bda", "fullName": "cardano_node_tests.tests.test_configuration#test_epoch_length", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('temp_dir')"}, {"name": "parentSuite", "value": "cardano_node_tests.tests"}, {"name": "suite", "value": "test_configuration"}, {"name": "host", "value": "bender-3900x"}, {"name": "thread", "value": "29225-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cardano_node_tests.tests.test_configuration"}], "links": [{"type": "link", "url": "https://github.com/input-output-hk/cardano-node-tests/blob/d28a503424b5f9f0359a9e79ea272dfac0f801bd/cardano_node_tests/tests/test_configuration.py#L110", "name": "https://github.com/input-output-hk/cardano-node-tests/blob/d28a503424b5f9f0359a9e79ea272dfac0f801bd/cardano_node_tests/tests/test_configuration.py#L110"}]}